core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -949,6 +949,93 @@ public void modeInUnsupportedFormat() {
         exception.getMessage());
   }
 
+  private void testWeekOfYear(String date, int mode, int expectedResult) {
+    FunctionExpression expression = DSL
+        .week_of_year(DSL.literal(new ExprDateValue(date)), DSL.literal(mode));
+    assertEquals(INTEGER, expression.type());
+    assertEquals(String.format("week_of_year(DATE '%s', %d)", date, mode), expression.toString());
+    assertEquals(integerValue(expectedResult), eval(expression));
+  }
+
+  private void testNullMissingWeekOfYear(ExprCoreType date) {
+    when(nullRef.type()).thenReturn(date);
+    when(missingRef.type()).thenReturn(date);
+    assertEquals(nullValue(), eval(DSL.week_of_year(nullRef)));
+    assertEquals(missingValue(), eval(DSL.week_of_year(missingRef)));
+  }
+
+  @Test
+  public void invalid_week_of_year() {
```

Comment:
can you make these `camelCase` names like `testInvalidWeekOfYear`

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/175/commits/17b888550c1068e21e5b94bc97e18309a8b2895c